oem/firstvoices/ios/FirstVoices/KeyboardsViewController.swift

```diff
@@ -0,0 +1,168 @@
+//
+//  KeyboardsViewController.swift
+//  FirstVoices app
+//
+//  License: MIT
+//
+//  Copyright © 2019 FirstVoices.
+//
+//  Created by Serkan Kurt on 17/11/2015.
+//  Converted and rewritten by Marc Durdin on 15/05/2019.
+//
+
+import UIKit
+import KeymanEngine
+
+let helpLink: String = "https://help.keyman.com/keyboard/"
+let tHelpButtonTag: NSInteger = 1000
+let tCheckBoxTag: NSInteger = 2000
+
+class KeyboardsViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
```

Comment:
I assume this is where the user selects which keyboards they want in their selectable listing?

Reply:
Yes.